tests/e2e/support/commands.ts

```diff
@@ -32,6 +32,51 @@
 // -- This is will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
 
+/// <reference types="cypress" />
+
+// Disable no-unused-vars because it is used in the return type of getStore
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
typescript should be smart enough for this; this error shouldnt happen. if it does, we can take a closer look at your eslint configuration

Reply:
Yeah, let's take a look at this later... Wasn't able to figure this out.